Package.swift

```diff
@@ -117,38 +124,61 @@ let package = Package(
       ]
     ),
     .testTarget(
-      name: "KSCrashRecordingCoreTests",
+      name: "\(Targets.recordingCore)Tests",
       dependencies: [
-        "KSCrashTestTools",
-        "KSCrashRecordingCore",
-        "KSCrashCore",
+        .target(name: Targets.testTools),
+        .target(name: Targets.recordingCore),
+        .target(name: Targets.core),
       ]
     ),
 
     .target(
-      name: "KSCrashReportingCore",
-      dependencies: ["KSCrashCore"]
+      name: Targets.reportingCore,
+      dependencies: [
+        .target(name: Targets.core)
+      ],
+      resources: privacyResource
     ),
     .testTarget(
-      name: "KSCrashReportingCoreTests",
+      name: "\(Targets.reportingCore)Tests",
       dependencies: [
-        "KSCrashReportingCore",
-        "KSCrashCore",
+        .target(name: Targets.reportingCore),
+        .target(name: Targets.core),
       ]
     ),
 
     .target(
-      name: "KSCrashCore"
+      name: Targets.core,
+      resources: privacyResource
     ),
     .testTarget(
-      name: "KSCrashCoreTests",
-      dependencies: ["KSCrashCore"]
+      name: "\(Targets.core)Tests",
+      dependencies: [
+        .target(name: Targets.core)
+      ]
     ),
 
     .target(
-      name: "KSCrashTestTools",
-      dependencies: ["KSCrashRecordingCore"]
+      name: Targets.testTools,
+      dependencies: [
+        .target(name: Targets.recordingCore)
+      ]
     ),
   ],
   cxxLanguageStandard: .gnucxx11
 )
+
+enum Targets {
+  static let recording = "KSCrashRecording"
+  static let filters = "KSCrashFilters"
+  static let sinks = "KSCrashSinks"
+  static let installations = "KSCrashInstallations"
+  static let recordingCore = "KSCrashRecordingCore"
+  static let reportingCore = "KSCrashReportingCore"
+  static let core = "KSCrashCore"
+  static let testTools = "KSCrashTestTools"
+}
+
+let privacyResource: [Resource] = [
```

Comment:
Let's either rename this to `privacyResources` or have it returning a single `Resource` object